resources/schema/USDM.yaml

```diff
@@ -0,0 +1,67 @@
+"`this`": "null"
```

Comment:
As the only situation in which `` `this` `` is passed to `entity_dict` is when there is no `context.path` for the attribute - i.e., only for the `study` attribute - should this be ``"`this`": Wrapper`` to align with the API spec?

Reply:
Updated